app/src/ui/lib/authentication-form.tsx

```diff
@@ -139,33 +138,28 @@ export class AuthenticationForm extends React.Component<
   }
 
   private renderSignInWithBrowser() {
-    const basicAuth = this.props.supportsBasicAuth
-    const browserSignInLink = (
-      <LinkButton
-        className="welcome-link-button link-with-icon"
-        onClick={this.signInWithBrowser}
-      >
-        Sign in using your browser
-        <Octicon symbol={OcticonSymbol.linkExternal} />
-      </LinkButton>
-    )
-
-    const browserSignInButton = (
-      <Button type="submit" onClick={this.signInWithBrowser}>
-        Sign in using your browser
-      </Button>
-    )
+    // we don't render this here because the user will have already
+    // had the option to sign in via the browser earlier in the sign-in flow
+    if (this.props.endpoint === getDotComAPIEndpoint()) {
+      return
+    }
 
     return (
-      <div>
-        {basicAuth ? <hr className="short-rule" /> : null}
-        {basicAuth ? null : this.renderEndpointRequiresWebFlow()}
-
-        <div className="sign-in-footer">
-          {basicAuth ? browserSignInLink : browserSignInButton}
-          {basicAuth ? null : this.renderActions()}
-        </div>
-      </div>
+      <>
+        {this.props.supportsBasicAuth && <hr />}
+        {this.props.supportsBasicAuth && this.renderEndpointRequiresWebFlow()}
```

Comment:
This was wrong, the previous code did the opposite 🤦‍♂️

Reply:
omg 🙀